### ISSUE TITLE
Function "read_data" problem

### ISSUE THREAD

**Issue by JJaudemar** [FedotovD/WESAD_ECG_SSL#2]
I have some question about the function "read_data".
The for loop is only read the "train_participants", so your train, test and validation dataset maybe are all the same ?

**Comment by almerbuda** [comment]
Yes JJaudemar is right. In the for loop use participants , not train_participants. The results are totally different then.
